src/Heatmap/HeatmapSeries/HeatmapSeries.tsx

```diff
@@ -54,6 +54,11 @@ export interface HeatmapSeriesProps {
    * Cell component that will be rendered.
    */
   cell: ReactElement<HeatmapCellProps, typeof HeatmapCell>;
+
+  /**
+   * Selected cell(s) in active state
+   */
+  selectedValues?: any;
```

Comment:
maybe we should call this `selectedPoints` to align with the `point` variable name in the helper functions

Reply:
or maybe `activePoints` to align even _more_ with `getColor()` params heh

Reply:
We have selection in other spots of the code - we should make the variables match that.

Reply:
Changed to `selections`.